### DISCUSSION TITLE
$env:POSH_GIT_ENABLED = $true in Powershell 7 disabled oh-my-posh

### DISCUSSION THREAD

**Discussion by seanadams9** [JanDeDobbeleer/oh-my-posh#4092] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Recently upgraded to Powershell 7 and noticed I wasn't seeing oh-my-posh anymore. Debugged and the $env:POSH_GIT_ENABLED = $true is disabling oh-my-posh somehow. 

In Powershell 7 if I type "$env:POSH_GIT_ENABLED = $true" and hit enter oh-my-posh goes away, then if I enter "$env:POSH_GIT_ENABLED = $false" oh-my-posh is back.

Confirmed in Powershell 5 I don't have this issue.

### Theme

Custom Theme

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
N/A
```

**Comment by JanDeDobbeleer** [comment]
@seanadams9 I can't reproduce this at all. I have oh-my-posh and posh-git (1.1.0) on latest. All this environment variable triggers is the following code:

```powershell
$global:GitStatus = Get-GitStatus
$env:POSH_GIT_STATUS = $global:GitStatus | ConvertTo-Json
```

So unless `Get-GitStatus` overwrites the prompt, this can never remove oh-my-posh.

**Reply by seanadams9** [reply]
Found the issue, PowerShell 7 modules are installed in a different location so posh-git was installed for PowerShell 5 but PowerShell 7 was looking in a different location. Wonder if there should be some error handling here though if Get-GitStatus fails.

**Reply by JanDeDobbeleer** [reply]
@seanadams9 we had that initially but that creates a significant performance hit. That's why we enable it with env vars explicitly.

**Comment by Steel-Shadow** [comment]
Pwsh7 and Windows PowerShell run in parallel. They have different modules.
I encountered the same issue. I installed posh-git on Windows PowerShell (5.1), but it was not installed on pwsh7.

Check if you have posh-git installed on pwsh7 and PowerShell, respectively. 
 ```pwsh
Get-InstalledModule
```

If you don't find posh-git, use `install-Module posh-git`.
